features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/textcomposer/MessageComposerPresenter.kt

```diff
@@ -48,6 +51,11 @@ class MessageComposerPresenter @Inject constructor(
             mutableStateOf(MessageComposerMode.Normal(""))
         }
 
+        val dmMember by room.getDmMember().collectAsState(initial = null)
```

Comment:
There will be some changes here cause I'm modifying a bit the api, but I'll handle them.